### ISSUE TITLE
Val IoU not good after training

### ISSUE THREAD

**Issue by arminbazza** [bradyz/cross_view_transformers#32]
Hey there, thank you for your great contribution! Unfortunately I am running into some issues when I am train the model using the nuscenes vehicle experiment config.

The iou metrics are as follows: 
| Train/Val | iou@0.4 | iou@0.5|
| --------- | ----------|----------|
| Train | 0.4305| 0.3868 |
| Val |  0.09296 | 0.03948 |

So, you can see that there are some issues during validation?

Interestingly, this can also be visually confirmed as during training the models outputs get better and better.
However, during validation the predictions get better, then there is a period where the model only predicts a black screen, and then they start getting better again.
<br>
Following are some photos from the w&b log:
Val metrics plot
![image](https://user-images.githubusercontent.com/29127715/193160232-07165fb8-2ff7-4bb7-8269-671c3f3337d8.png "val metrics")

Val output at step 247
![image](https://user-images.githubusercontent.com/29127715/193161109-0ec144f9-f052-461e-91fa-085cc5014ca6.png)

Val output at step 329 (notice all black)
![image](https://user-images.githubusercontent.com/29127715/193161002-7930ad97-a82d-48fe-9b74-b89d85c53414.png)

Val output at step 663
![image](https://user-images.githubusercontent.com/29127715/193161061-78342c13-1b4e-4f5b-bc5e-af85da71af1d.png)

Would really appreciate any feedback you might have regarding this issue. Thanks.

**Comment by gsg213** [comment]
Hello @ArminBaz ,

Did you find the issue?
Did you change something in the model architecture?

For further analysis, you could share the loss and lr graphs to understand better what happened there.

**Comment by arminbazza** [comment]
Hey @gsg213, I did end up resolving the issue on my own. 

What worked for me was lowering the learning rate inside of `config.yaml` from 4e-3 to 1e-3. 

Hope this helps!
